### DISCUSSION TITLE
confused by openldap formula

### DISCUSSION THREAD

**Discussion by kevwil** [Homebrew/discussions#3250] [Everyday usage] [answered]
Installing / upgrading openldap on my M1 mbp shows this message:
```
openldap is keg-only, which means it was not symlinked into /opt/homebrew,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.
```
However, it's not on the path:
```
❯ which -a openldap
openldap not found
❯ echo $PATH
/Users/keviwill/.rbenv/shims:/opt/homebrew/opt/python@3.9/libexec/bin:/Users/keviwill/.asdf/shims:/opt/homebrew/opt/asdf/libexec/bin:/opt/homebrew/opt/mysql@5.7/bin:/Users/keviwill/.jenv/shims:/Users/keviwill/go/bin:/Users/keviwill/.krew/bin:/Users/keviwill/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/usr/local/sbin:/opt/homebrew/bin:/Users/keviwill/gradle/bin
```
I'm confused why this formula is keg-only. Perhaps it is some other conflict besides the main executable? Perhaps I need more caffeine? ;-)

**Comment by SMillerDev** [comment, marked as answer]
It's there on my machine, the command just isn't openldap. You can check `man slapd` or `/usr/libexec/slapd`

**Comment by kevwil** [comment]
LOL I think the caffeine was the problem. I needed the `ldapsearch` tool that is already provided, no idea why I thought `openldap` was the executable. Stupid Mondays.
